locale/zh_CN.po

```diff
@@ -2094,94 +2095,94 @@ msgstr "无法从文件 '%s' 中读取图像 %%d。"
 #: ../src/common/iconbndl.cpp:206
 #, c-format
 msgid "Failed to load image %d from stream."
-msgstr "无法从数据流中载入图像 %d。"
+msgstr "无法从数据流中加载图像 %d。"
 
 #: ../src/common/iconbndl.cpp:224
 #, c-format
 msgid "Failed to load icons from resource '%s'."
-msgstr "无法从资源”%s”中载入图标 。"
+msgstr "无法从资源 \"%s\" 中加载图标 。"
 
 #: ../src/common/imagbmp.cpp:97
 msgid "BMP: Couldn't save invalid image."
-msgstr "BMP: 无法保存无效图像。"
+msgstr "BMP：无法保存无效图像。"
 
 #: ../src/common/imagbmp.cpp:137
 msgid "BMP: wxImage doesn't have own wxPalette."
-msgstr "BMP: wxImage 没有自己的 wxPalette。"
+msgstr "BMP：wxImage 没有自己的 wxPalette。"
 
 #: ../src/common/imagbmp.cpp:243
 msgid "BMP: Couldn't write the file (Bitmap) header."
-msgstr "BMP: 无法写文件头 (Bitmap)。"
+msgstr "BMP：无法写文件 (Bitmap) 表头。"
 
 #: ../src/common/imagbmp.cpp:266
 msgid "BMP: Couldn't write the file (BitmapInfo) header."
-msgstr "BMP: 无法写文件头 (BitmapInfo)。"
+msgstr "BMP：无法写文件 (BitmapInfo) 表头。"
 
 #: ../src/common/imagbmp.cpp:353
 msgid "BMP: Couldn't write RGB color map."
-msgstr "BMP: 无法写 RGB 色彩表。"
+msgstr "BMP：无法写 RGB 色彩表。"
 
 #: ../src/common/imagbmp.cpp:487
 msgid "BMP: Couldn't write data."
-msgstr "BMP: 无法写数据。"
+msgstr "BMP：无法写数据。"
 
 #: ../src/common/imagbmp.cpp:533
 #, c-format
 msgid "BMP: header has biClrUsed=%d when biBitCount=%d."
-msgstr "BMP: 当 biBitCount=%d 时，文件头为 biClrUsed=%d。"
+msgstr "BMP：当 biBitCount=%d 时，表头則為 biClrUsed=%d。"
```

Comment:
```suggestion
msgstr "BMP：当 biBitCount=%d 时，表头则为 biClrUsed=%d。"
```

Reply:
fixed in .po

Reply:
Found something weird... wondered why zh_CN.po leverage zh_TW.po after updating from .pot file.
![Screen Shot 2022-07-11 at 10 55 29](https://user-images.githubusercontent.com/82932766/178180612-8a5044ab-b335-4148-ae9e-62731f25effc.png)